lib/ansible/cli/doc.py

```diff
@@ -1351,17 +1351,6 @@ def get_role_man_text(self, role, role_json):
                     # use empty indent since this affects the start of the yaml doc, not it's keys
                     text.append(DocCLI._indent_lines(DocCLI._dump_yaml({k.upper(): doc[k]}), ''))
 
-            if doc.get('examples', False):
-                text.append('')
-                text.append(_format("EXAMPLES:", 'bold'))
-                if isinstance(doc['examples'], string_types):
-                    text.append(doc.pop('examples').strip())
-                else:
-                    try:
-                        text.append(yaml_dump(doc.pop('examples'), indent=2, default_flow_style=False))
-                    except Exception as e:
-                        raise AnsibleParserError("Unable to parse examples section", orig_exc=e)
-
```

Comment:
Please re-add examples. They were explicitly added as a feature in #82671.

Reply:
sorry, will do, had removed since it was not in the original spec and I could not find a use of it nor make sense of a scenario in which adding an example of using an argspec (really just importing the role) made sense.